libs/language-server/src/lib/ast/expressions/test-utils.ts

```diff
@@ -45,10 +46,14 @@ export async function executeExpressionTestHelper(
         }
     `);
 
-  const transform = locator.getAstNode<TransformDefinition>(
-    document.parseResult.value,
-    'transforms@0',
-  ) as TransformDefinition;
+  const allElements = AstUtils.streamAllContents(document.parseResult.value);
+  const allTransforms = [...allElements.filter(isTransformDefinition)];
+  expect(
+    allTransforms.length !== 0,
```

Comment:
Test and comment are at adds, test is true for 2, comment wants exactly 1.

Reply:
Nice catch! That line is actually wrong like this and does nothing. Fixed it in [3db1d0e](https://github.com/jvalue/jayvee/pull/571/commits/3db1d0ea9fced4a107c23d1fe0e0562b2ed523eb)